### PR TITLE
Task/30 Create EmailValidator

### PR DESCRIPTION
**Pull request by rafalkrystek**

### CODE REVIEW

**Review by przemek19980102 — CHANGES_REQUESTED** [review]
Please rebase onto master and take a look at PhoneNumberValidator. Your class should have similar methods

File: app/src/main/java/com/safetyheads/akademiaandroida/utils/EmailValidator.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+
+public class EmailValidator extends AppCompatActivity {
+
+    public void emailValidator(EditText etMail) {
+        String emailToText = etMail.getText().toString();
+        if (!emailToText.isEmpty() && Patterns.EMAIL_ADDRESS.matcher(emailToText).matches()) {
```
Comment (msarelo) [inline app/src/main/java/com/safetyheads/akademiaandroida/utils/EmailValidator.java:14]: It's not something we can reuse in other parts in the app.

**Review by msarelo — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_launch_screen.xml [hunk 2]
```diff
@@ -76,7 +76,7 @@
 
     <Button
         android:id="@+id/button"
-        style="@style/App_button"
+        style="@style/Button"
```
Comment (przemek19980102) [inline app/src/main/res/layout/fragment_launch_screen.xml:79]: Are you sure those changes should be in this PR?

File: app/src/main/res/layout/fragment_we_are_hiring.xml [hunk 3]
```diff
@@ -10,7 +9,7 @@
 
     <Button
         android:id="@+id/apply_now"
-        style="@style/App_button"
+        style="@style/Button"
```
Comment (przemek19980102) [inline app/src/main/res/layout/fragment_we_are_hiring.xml:12]: Are you sure those changes should be in this PR?

**Review by przemek19980102 — CHANGES_REQUESTED** [review]

**Review by msarelo — APPROVED** [review]

**Review by przemek19980102 — APPROVED** [review]

**Pull request merged**